dll/win32/msi/dialog.c

```diff
@@ -3160,7 +3160,19 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
             msi_free( num );
+#ifdef __REACTOS__
+            if ((count == 0) && (wcsncmp( num, L"\\", 1 ) || wcsncmp( num, L"&", 1 )))
```

Comment:
The condition `(wcsncmp( num, L"\\", 1 ) || wcsncmp( num, L"&", 1 ))` is always true, so effectively this code is back to the first iteration where anything invalid is ignored.

You probably meant `(!wcsncmp( num, L"\\", 1 ) || !wcsncmp( num, L"&", 1 ))`

Reply:
Moreover, why is `wcsncmp()` needed, to compare a single character?